server/src/main/scala/za/co/absa/atum/server/model/PartitioningForDB.scala

```diff
@@ -32,4 +33,7 @@ object PartitioningForDB {
 
     PartitioningForDB(keys = allKeys, keysToValuesMap = mapOfKeysAndValues)
   }
+
+  implicit val writes: Writes[PartitioningForDB] = Json.writes
```

Comment:
hmm potentially we might need reads as well, relatively soon?

Reply:
We will always need both reads/writes for endpoints. Here we need only writes as we don't receive this via rest call but rather construct ourselves.